packages/components/src/components/scroll-buttons/scroll-buttons.tsx

```diff
@@ -5,12 +5,32 @@ import { Component, h, JSX, Method, Prop, Watch } from '@stencil/core';
     styleUrl: 'scroll-buttons.scss',
     shadow: true
 })
+
 export class ScrollButtons {
     /**
      * Reference to the element with scroll on parent element.
+     *
      * @type {HTMLDivElement}
      */
     @Prop() scrollContainerElementRef: HTMLDivElement;
+
+    /**
+     * Items that are rendered inside a specific element.
+     */
+    @Prop() elementItems;
```

Comment:
I don't see elementItems being set from outside of the component. If this is internal only, it should use a `@state` decorator.

Reply:
Also, is it even needed? Can't we calculate the number of child elements when we need it?

Reply:
I used `@State`. I was thinking to keep it as separate variable, so it is easier to understand what it is. @ammapspeople

Reply:
But is it needed? We are forcing the consumer of the component to add additional info to the method call updateScrollButtons. It seems as the scroll buttons component should already have access to it (using some variant of querySelector, or by other means).